### ISSUE TITLE
body-parser error results in abrupt connection close

### ISSUE THREAD

**Issue by noqcks** [rayepps/exobase-js#16]
when using exbobase w/ express and sending in a malformed json body to a POST

(extra comma at the end of mypayload) 
```
{
    "mybody": {
        "name": "mypayload", 
    }
}
```

the server abruptly terminates
```
SyntaxError: Expected double-quoted property name in JSON at position 103 (line 4 column 5)
    at JSON.parse (<anonymous>)
    at parse (/app/node_modules/.pnpm/body-parser@1.20.1/node_modules/body-parser/lib/types/json.js:89:19)
    at /app/node_modules/.pnpm/body-parser@1.20.1/node_modules/body-parser/lib/read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (/app/node_modules/.pnpm/raw-body@2.5.1/node_modules/raw-body/index.js:231:16)
    at done (/app/node_modules/.pnpm/raw-body@2.5.1/node_modules/raw-body/index.js:220:7)
    at IncomingMessage.onEnd (/app/node_modules/.pnpm/raw-body@2.5.1/node_modules/raw-body/index.js:280:7)
    at IncomingMessage.emit (node:events:519:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12) {
  expose: true,
  statusCode: 400,
  status: 400,
  body: '{\n' +
    '    "mybody": {\n' +
    '        "name": "mypayload",\n' +
    '    }\n' +
    '}',
  type: 'entity.parse.failed'
}
Node.js v21.5.0
 ELIFECYCLE  Command failed with exit code 1.
/app/apps/component-api:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @xeol/component-api@0.1.0 start: `pnpm with-env tsx src/server.ts`
Exit status 1
```

in postman I see
```
Could not send request: Error: connect ECONNREFUSED 127.0.0.1:3000
```

the line throwing is
```
  const requestAfterMiddlware = await middleware(req, res)
```

which I can get to not fail with a try/catch

I've also tried to catch by adding a top level express error handler but logic never reaches this far before blowing up
```
app.use((err: Error, req: Request, res: Response) => {
    ...
});
```

**Comment by rayepps** [comment]
Thanks for bringing this up @noqcks, it absolutely should not crash the server in that case 🙃 The issue has been fixed ([ccf6c30d](https://github.com/rayepps/exobase-js/commit/ccf6c30da85fdcda428316cccb716a1e36de8ec5)) and published in `@exobase/use-express` to npm.

To provide more context, and potentially help others, there is still a fundamental issue here with express middleware. The json body parsing and compression middleware are built into the `useExpress` hook and get run within the `useExpress` function. It's an issue because if there's an error you can't access it. I decided to include them when I wrote it because they're so fundamental to HTTP, it would be unexpected to devs IMHO if they had to wire those up themselves. 

However, I did build an escape hatch here, I just didn't document it. The `useExpress` [ hook's options](https://github.com/rayepps/exobase-js/blob/b5eee90e52e0db38db632ee1226a966b3d819699/packages/use-express/src/useExpress.ts#L28) allow you to turn off the middleware. So, if you wanted you could skip the Express middleware and do the work yourself, handling errors any way you need to.

```ts
useExpress({
    skipJson: true,
    skipCompression: true
})
```

**Comment by noqcks** [comment]
Thanks for the quick fix! @rayepps
